### ISSUE TITLE
Ore Decorations not appearing

### ISSUE THREAD

**Issue by clubpetey** [superckl/BiomeTweaker#215]
BT verison: 3.2.354
Forge: 2795

I created a custom dimension (42) using the customized generator and JED with all ore-gen turned off.
I then added ores back in with the script below.

https://pastebin.com/aEZVuhk4

The tree decorations do appear in the world, but the ore decorations do not.  

*Possibly related* I also removed and attempted to replace ore-gen in the overworld using the following script.  

https://pastebin.com/DstMqLyb

Here also, no ore decorations generate.  The immersive engineering ores throw an error, due to the meta-tag (will open a separate ticket for that), but I would expect the vanilla ores to generate, they do not.

**Comment by superckl** [comment]
How have you disabled ore gen through JED? My guess would be that it is simply disabling the ore gen event, and thus BT doesn't see the ore gen event. Try using a placement stage that doesn't mention "ORES".

**Comment by clubpetey** [comment]
I took JED out to see if it was causing problems, even in a standard overworld I was not getting any ore decoration.  But I found the issue, I MUST call `addBlockToReplace()` as part of my decoration setup, even though the docs say this is defaulted to stone.

Perhaps the default isn't correct or some other mod is messing with stone, but adding :
`dec.addBlockToReplace("minecraft:stone")`

to each ore-gen decoration fixed the issue.
